### DISCUSSION TITLE
The lift model in Webots

### DISCUSSION THREAD

**Discussion by ZiqiTIAN** [cyberbotics/webots#3667] [Q&A]
Hi, I am a beginner of  using Webots simulator. I checked the environment and found that there is only stairs in the enviroment instead of lift. I thought that I could creat a robot with lift apperance and slider joints, then worte an controller to control its movment. Is there any ideas that could build the lift in a much easier way? And I have no idea about how to simulate the press button for the lift. I hope hear others' ideas.

**Comment by lukicdarkoo** [comment]
> I thought that I could creat a robot with lift apperance and slider joints, then worte an controller to control its movment. Is there any ideas that could build the lift in a much easier way?

That sounds like a good idea.

> And I have no idea about how to simulate the press button for the lift.

You can use the `TouchSensor`:
https://cyberbotics.com/doc/reference/touchsensor

P.S. I converted the issue to a discussion as it is rather a question than a bug/feature suggestion.

**Reply by ZiqiTIAN** [reply]
Thank you very much!
